examples/tdengineTelemetryService/deployment/telemetryservice-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
+  password: cmVhbF90YW9zZGF0YQ==
```

Comment:
Let's replace this field with "REDACTED"

Reply:
ok